devtools/conda-envs/docs_env.yaml

```diff
@@ -1,5 +1,7 @@
 name: openff-interchange-docs
 channels:
+  # Skip installing CUDA to speed up builds
+  - jaimergp/label/unsupported-cudatoolkit-shim
```

Comment:
👍

(though obviously we'll eventually want the fundamental OpenMM-no-CUDA problem fixed upstream so we're not using unsupported things in production!)